packages/example-app-base/src/components/ComponentStatus/ComponentStatus.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { IComponentStatusProps, TestCoverageStatus } from './ComponentStatus.Props';
+import './ComponentStatus.scss';
+
+export class ComponentStatus extends React.Component<IComponentStatusProps, {}> {
+
+  public static defaultProps: IComponentStatusProps = {
+    keyboardAccessibilitySupport: false,
+    markupSupport: false,
+    highContrastSupport: false,
+    rtlSupport: false,
+    testCoverage: TestCoverageStatus.none
+  };
+
+  public render(): JSX.Element {
+    const keyboardAccessibilitySubject = 'Keyboard Accessibility';
+    const markupSubject = 'Markup';
+    const highContrastSupportSubject = 'High Contrast';
+    const rtlSubject = 'Right to Left';
+    const testCoverageSubject = 'Test Coverage';
+
+    return (
+      <div className='ComponentStatus-div'>
+        { this._badgeAnchor('Keyboard accessibility support. ' +
+          this._badgeStatusString(this.props.keyboardAccessibilitySupport), this._badgeURL(
+            this._colorForDualStatusBadge(this.props.keyboardAccessibilitySupport),
+            keyboardAccessibilitySubject,
+            this._badgeStatusString(this.props.keyboardAccessibilitySupport))) }
```

Comment:
Why not move all of these function calls inside of the badgeAnchor function and just pass the label prefix, subject, and the prop. Doing that will make this look a lot cleaner and will make it to where all of those function calls will only be inside of badgeAnchor